save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/vulnerability/VulnerabilityStatus.kt

```diff
@@ -10,17 +10,17 @@ import kotlinx.serialization.Serializable
  */
 @Serializable
 @JsExport
-@Suppress("IDENTIFIER_LENGTH")
+@Suppress("IDENTIFIER_LENGTH", "WRONG_DECLARATIONS_ORDER")
 enum class VulnerabilityStatus(val value: String) {
     /**
-     * Approve status
+     * Automatically approved by batch-upload
      */
-    APPROVED("Approved"),
+    AUTO_APPROVED("Uploaded"),
```

Comment:
why uploaded?

Reply:
Only vulnerabilities uploaded using .json files get `AUTO_APPROVED` status, while they don't go through `Created` stage and do not require to be `APPROVED` from admin. Therefore, for better understanding, I decided to describe such vulnerabilities with `UPLOADED` status.

Reply:
I think better to leave auto approved. @akuleshov7